assets/dev/js/editor/controls/color.js

```diff
@@ -62,6 +65,19 @@ export default class extends ControlBaseDataView {
 		jQuery( this.colorPicker.picker.getRoot().root ).addClass( 'elementor-control-unit-1 elementor-control-tag-area' );
 	}
 
+	hidePickerOnPreviewClick() {
+		const picker = this.colorPicker.picker;
+		const pickerUtils = picker.constructor.utils;
+
+		picker._eventBindings.push(
+			pickerUtils.on( elementorFrontend.elements.window.document, [ 'touchstart', 'mousedown' ], () => {
```

Comment:
Feels weird to bind events to a picker like that, is another way exists? when does it unBind the events?

Reply:
Unfortunately there is no method in the picker class that allows you to add to the eventBindings, have to do it like this.
I specifically wanted to add to the eventBindings array of the picker rather than just directly add an eventListener for example because in its destroy it unbinds all its events. The picker's destroy is called in color.js onBeforeDestroy()